src/components/Radio.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <fieldset class="f-radio">
+    <input
+      v-model="newResourceType.abstract"
```

Comment:
What's about the `newResourceType` here?

Reply:
Deleted the whole file.